src/me/mrCookieSlime/Slimefun/SlimefunGuide.java

```diff
@@ -713,8 +713,8 @@ public void run(final Player p) {
 							index++;
 						}
 						else {
-							List<String> tooltip = Messages.local.getTranslation("tooltips.item-permission");
-						    menu.addItem(index, new CustomItem(Material.BARRIER, StringUtils.formatItemName(sfitem.getItem(), false), tooltip.toArray(new String[tooltip.size()])));
+							List<String> message = SlimefunItem.getNoPermissionTooltip(sfitem.getItem());
```

Comment:
Oh god no.
Why would you loop through all Slimefun items to find the Slimefun Item that you already had...???

Just do sfItem.getNoPermissionTooltip() ...

Reply:
Yeah I was tired when I did that, fixed ;)